### DISCUSSION TITLE
A new chapter?

### DISCUSSION THREAD

**Discussion by AveYo** [No-Bling/DOTA#87] [Announcements]
Are we TF2 level yet? Upcoming 2023 BP giving anyone PTSD? It's been too long. But not forgotten..

**Comment by staaandup** [comment]
Big if true

**Comment by kaaiuta** [comment]
Used this mod for like 3-4 years when gaming on my old laptop. Just wanted to express my gratitude for doubling my FPS at that troubling time. I dont necessarily need this mod anymore, but it would be cool to be able to play "vanilla" dota every once in a while. Peace.
